src/HomePage/Skills/Skills.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./Skills.scss";
+
+const icons = [
+  "icons devicon-github-plain",
```

Comment:
If i was being very nitpicky i would say putting icons in everytime is repeating still and you could improve this. But i will leave it to you to decide if you want to do that, it's good enough though